lib/router.js

```diff
@@ -45,18 +46,26 @@ router.get('/slashCommands/ci', (req, res) => {
 router.get('/slashCommands/ext', (req, res) => {
     const command = req.query && req.query.command;
     const text = req.query.text;
+    const update = req.query.update;
```

Comment:
This is interesting... It involves the slack side, where you register the slack-cutter web service. I don't know exactly how it works but i'm guessing you can add another parameter if you want. Am I wrong? If I'm right the user would be able to request an update and at the same time make a search, which is cool. But I could make it work your way if we have to.